### ISSUE TITLE
[SUGGESTION] Auto-loadout swap

### ISSUE THREAD

**Issue by sebastienwood** [pokeclicker/pokeclicker#2196]
Always apply the bonus for the oak items for mine and farm.

**Current Behavior**
Some oak items are only applying to specific game features (farm, mine). This makes the repetitive process to:
- change loadout
- go to game feature
- exit, change loadout

**Improved Behavior**
As long as there are < 3 oak items that apply to farm, mine independently, let them always stay "on", giving bonus without having to swap loadout for a few seconds and not counting in the current 3 items limit.

**Reason for change**
This would remove a tedious micro-task without giving too much of a bonus to players.

**Detailed Description**
![image](https://user-images.githubusercontent.com/18282449/169409968-14a04519-3dc5-4914-a97c-69ea9480644b.png)
Is reworked. Last line is excluded from the picks, items removed are now in "Key items" tabs of the Item menu. Cell battery is not excluded as it's a global bonus (i.e. you needn't be mining for it to be an interesting item).

**Comment by Ultima1990** [comment]
Member when we didn't have loadouts at all?
Give a hand, they want the arm.

What even is the point of limited oak item slots and loadouts if they auto swap anyway?

**Comment by InQuognito** [comment]
Why don't instead, we give every Oak item a way to be interesting at any given point?

Sprayduck should actually make the timer move slightly faster, instead of a decreased time on planted berries. This makes it much more useful as an item to take up a slot, creating more strategy.

Explosive Charge is... useless, but its rework has been mentioned before by other users here and I genuinely like what they said.
Every x minutes it can essentially activate a bomb for free, adding a way to "idle" the Underground essentially. Obviously very slow (every 30 minutes at base probably)

Treasure Scanner would then also have a use without having to change its functionality.

The other two watering cans are fine as-is, since farm is automatable and these have valid functions in any given moment.

**Comment by Kays** [comment]
> Why don't instead, we give every Oak item a way to be interesting at any given point?
> 
> Sprayduck should actually make the timer move slightly faster, instead of a decreased time on planted berries. This makes it much more useful as an item to take up a slot, creating more strategy.
> 
> Explosive Charge is... useless, but its rework has been mentioned before by other users here and I genuinely like what they said. Every x minutes it can essentially activate a bomb for free, adding a way to "idle" the Underground essentially. Obviously very slow (every 30 minutes at base probably)
> 
> Treasure Scanner would then also have a use without having to change its functionality.
> 
> The other two watering cans are fine as-is, since farm is automatable and these have valid functions in any given moment.

I don't think that will change much, if anything. We can only have 3 Oak Items equipped at any given time. Making the Oak Items have passive abilities when they're equipped aren't enough to make me use them. Blaze Cassette and Shiny Charm are permanents; they're way too strong to not be equipped. Maybe Shiny Charm can go if you've done with Shiny 'dex, but still, there's the Shiny quest for QP for Justice/Carey. The last spot is usually Magic Ball unless I'm doing gym or Battle Frontier.

**Comment by Qwertypop04** [comment]
Closing. Having any Oak Items be "always on" defeats the entire purpose of them being Oak Items in the first place. If you think certain items should be buffed, that's a separate issue
